styles.css

```diff
@@ -92,6 +93,206 @@ h2 {
   justify-content: center;
 }
 
+.grid-container {
+  display: grid;
+  grid-column: auto;
+  grid-row: auto;
+  gap: 24px;
+  place-items: center;
+}
+
+.work h2 {
+  width: 327px;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #172b4d;
+  text-align: center;
+  align-items: center;
+  margin: 0 auto 24px auto;
+}
+
+.indicator {
+  align-items: center;
+  border-radius: 24px;
+  background: #36b37e;
+  width: 48px;
+  height: 4px;
+  margin: 0 auto 60px auto;
+}
+
+.work-1,
+.work-2,
+.work-3,
+.work-4,
+.work-5,
+.work-6 {
+  display: flex;
+  flex-direction: column;
+  justify-content: end;
+  border: 1px solid #d0d9d4;
+  border-radius: 8px;
+  background: #ebf0ee;
+  width: 327px;
+  height: 474px;
+}
+
+.project {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+  background: #fff;
+  border-radius: 8px;
+  width: 325px;
+  height: 222px;
+  padding: 16px;
+  text-align: center;
+}
+
+.project h3 {
+  font-size: 24px;
+  font-style: normal;
+  font-weight: 600;
+  line-height: 32px;
+  text-align: center;
+  color: #3a4a42;
+}
+
+.prog-lan {
+  align-items: baseline;
+  display: flex;
+  column-gap: 5px;
+  list-style: none;
+}
+
+.prog-lan li {
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 600;
+  line-height: 16px;
+  text-align: center;
+  color: #3a4a42;
+  background: #ebf0ee;
+  border-radius: 4px;
+  padding: 8px 12px;
+}
+
+.btn {
+  width: auto;
+  padding: 12px;
+  background: #36b37f;
+  border-radius: 4px;
+  font-style: normal;
+  font-weight: 300;
+  font-size: 17px;
+  line-height: 24px;
+  color: #fff;
+  border-style: none;
+  letter-spacing: 0.03em;
+}
```

Comment:
Please kindly implement the active, pressed and hover effects of all the buttons. It is true that these effects are not seen on mobile versions but it is a must to implement them at this stage of the project :+1:  since it is part of the project requirements as shown below (_check the last bullet point_ ) You can check the desktop side of the figma design for the button effects. :pray:
![Screenshot from 2022-09-23 16-24-44](https://user-images.githubusercontent.com/65233860/192025380-b449640b-960c-4b98-b345-0024634bf8b5.png)